rebar.config

```diff
@@ -84,7 +85,8 @@
         {extended_start_script_extensions, [
               {check_config, "extensions/check_config"},
               {keys_gen, "extensions/keys_gen"},
-              {peer_key, "extensions/peer_key"}
+              {peer_key, "extensions/peer_key"},
+              {export, "extensions/export_chain"}
```

Comment:
May I suggest it has a smoke test among [system tests](https://github.com/aeternity/epoch/blob/2c7db84032756f8c9eb28c7fcc934b04fc6fa590/system_test/aest_commands_SUITE.erl)?

Reply:
@lucafavatella a smoke test is not simple without an import command. I would need to implement the import command in some way in the test. My idea was to add a test when implementing the import, but if you think it must have one I could add one. There would be two options, mocking the chain or doing a system test, exporting the chain and then reading the file in the test itself assuming its format.